### ISSUE TITLE
Script doesn't detect when the stream drops/returns instantly, or from passworded shows

### ISSUE THREAD

**Issue by MeatyChunks** [jrudess/mfc_cb.node#24]
With Chaturbate streams it seems like the script doesn't notice when the streamer quickly restarts their stream so no new output file is created, nor is anything written to the current one. There was also a channel tonight that got set to passworded while they were public but the script didn't notice, even when they returned. I should note I changed `modelScanInterval` to 30 seconds because some of the streamers I follow go into privates quite often, would this affect it at all?

Really appreciate the work you've put into this though, since other methods like streamlink put out files with constant pauses.

**Comment by MeatyChunks** [comment]
It also seems like it just fails to finish sometimes too. Tonight I left it running for one streamer and two hours later it still says already capturing.
![image](https://user-images.githubusercontent.com/6160351/27884743-410f90a2-61cd-11e7-8c67-88f57debf63f.png)

I press ctrl+c to forcibly end it, it will say waiting for capture stream to end and never progress until ctrl+c is pressed again. Then it'll finally convert the file.

**Comment by MeatyChunks** [comment]
Having tried both master and dev branches now, dev does seem a little better at noticing. However, it seems like the script still stops writing data to the output file and doesn't seem to explain why in the console. It just keeps saying it's already capturing when it isn't writing new data any more.

**Comment by jrudess** [comment]
Yes, a lower modelScanInterval should improve this, because the script will auto terminate any ffmpeg process active when it detects that a model is not streaming anymore.   

The problem mostly happens when a model disconnects and reconnects within that scan interval, so the script never sees the model as offline.   But, smaller intervals do mean fetching pages more often from CB, so just be aware of the possibility of IP banning.  

For MFC, the interval doesn't impact how fast or slow MFC updates occur, as a different library (MFCAuto) is handling that asynchronously.
